test/lein_monolith/task/with_dependency_set_test.clj

```diff
@@ -0,0 +1,36 @@
+(ns lein-monolith.task.with-dependency-set-test
+  (:require
+    [clojure.test :refer [deftest testing is]]
+    [lein-monolith.task.with-dependency-set :as wds]
+    [lein-monolith.test-utils :refer [use-example-project]]
+    [leiningen.core.main :as lein]
+    [leiningen.core.project :as project]))
+
+
+(use-example-project)
+
+
+(defn- read-project
+  [path]
+  (project/read path))
```

Comment:
Is this useful? `read-project` and `project/read` have the same number of characters, even.